code/ai/ai_profiles.cpp

```diff
@@ -379,6 +379,9 @@ void parse_ai_profiles_tbl(const char *filename)
 				if (optional_string("$Detail Distance Multiplier:"))
 					parse_float_list(profile->detail_distance_mult, MAX_DETAIL_VALUE + 1);
 
+				if (optional_string("$Max Allies Rearming Threshold:"))
```

Comment:
Super minor, but the more modern and single value style flags overall use lower-case. Also, you might consider having the location of this new flag be near/right after the '$support don't add primaries:'  line, since that flag also relates to support values. :)